llvm/lib/CodeGen/PreISelIntrinsicLowering.cpp

```diff
@@ -394,11 +394,12 @@ bool PreISelIntrinsicLowering::expandMemIntrinsicUses(Function &F) const {
       Module *M = Memset->getModule();
       const DataLayout &DL = Memset->getDataLayout();
 
+      Type *DestPtrTy = Memset->getRawDest()->getType();
+      Type *IntIdxTy = TLI.getSizeTType(*M);
```

Comment:
```suggestion
      Type *SizeTy = TLI.getSizeTType(*M);
```
might be a bit clearer now...